util/patch/patch_test.go

```diff
@@ -365,7 +367,7 @@ func TestPatchHelper(t *testing.T) {
 
 				objCopy := obj.DeepCopy()
 
-				t.Log("Marking a custom condition to be false")
+				t.Log("Marking ")
```

Comment:
Looks like sed went wrong, same in l.425, 474

Reply:
fixed